scripts/build_layers.sh

```diff
@@ -11,6 +11,13 @@ set -e
 LAYER_DIR=".layers"
 LAYER_FILES_PREFIX="datadog_lambda_node"
 NODE_VERSIONS=("10.15" "12.13")
+DATADOG_LAMBDA_VERSION=$(cat package.json | jq -r ".version")
+DD_TRACE_VERSION=$(cat package.json | jq -r ."devDependencies"'."dd-trace"')
```

Comment:
The version of dd-trace-js specified in the package.json can be just a suggestion. For instance ^0.20.0, would mean any minor version greater than 20, but not a major version, so 0.25.5 would match, but not 1.21.0. Right now it's locked in the package.json to an explicit version, but we might change that in the future. The place to actually search is the yarn.lock file, which usually has an entry like this: https://github.com/DataDog/datadog-lambda-js/blob/master/yarn.lock#L1341 . I think we can leave it for now though.

Reply:
Done!